Yoast/wordpress-seo
# Show a notice to users running PHP5.2

## Summary

This PR can be summarized in the following changelog entry:
* Adds a notice about outdated PHP5.2 versions

## Relevant technical choices:

Using Whip: Show a notice to users not running PHP5.3 or higher. We can
easily increase this later.

## Test instructions

This PR can be tested by following these steps:

* Run `composer install`
* Go to PHP5.2
* You should see a notice in the admin informing you that your PHP version is outdated.

Fixes #6293
